public/css/schema-editor.css

```
@@ -0,0 +1,18 @@
+/* Error: Undefined mixin.
+ *    ,
+ * 22 | /       @include tablet-down {
+ * 23 | |         flex-direction: column-reverse;
+ * 24 | \       }
+ *    '
+ *   src/styles/schema-editor.scss 22:7  root stylesheet */
+
+body::before {
+  font-family: "Source Code Pro", "SF Mono", Monaco, Inconsolata, "Fira Mono",
+      "Droid Sans Mono", monospace, monospace;
```

Biome (1.9.4)

[error] Duplicate font names are redundant and unnecessary: monospace

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)